example/standalone-user-provided-creds/main.tf

```
@@ -1,30 +1,41 @@
 module "minio" {
-  source = "github.com/fredrikhgrelland/terraform-nomad-minio.git?ref=0.2.0"
+  source = "github.com/fredrikhgrelland/terraform-nomad-minio.git?ref=0.3.0"
 
   # nomad
   nomad_datacenters = ["dc1"]
   nomad_namespace   = "default"
   nomad_host_volume = "persistence-minio"
 
   # minio
-  service_name                    = "minio"
-  host                            = "127.0.0.1"
-  port                            = 9000
-  container_image                 = "minio/minio:latest" # todo: avoid using tag latest in future releases
-  access_key                      = "minio"
-  secret_key                      = "minio123"
+  service_name    = "minio"
+  host            = "127.0.0.1"
+  port            = 9000
+  container_image = "minio/minio:latest" # todo: avoid using tag latest in future releases
+  # user provided  credentials
+  vault_secret = {
+    use_vault_provider   = false,
+    vault_kv_policy_name = "",
+    vault_kv_path          = "",
+    vault_kv_access_key    = "",
+    vault_kv_secret_key    = ""
+  }
+  access_key = "minio"
+  secret_key = "minio123"
+
   data_dir                        = "/minio/data"
   buckets                         = ["default", "hive"]
   container_environment_variables = ["JUST_EXAMPLE_VAR1=some-value", "ANOTHER_EXAMPLE2=some-other-value"]
-  use_host_volume                 = true
+  use_host_volume                 = false
+  use_canary                      = false
```

Didn't we talk about having all features enabled, or should that be in another example?
```suggestion
  use_host_volume                 = true
  use_canary                      = true
```

---

fixed